### PR TITLE
Move MVO to using build-root from_repository

### PR DESCRIPTION
**Pull request by cblecker**
Follow up to https://github.com/openshift/managed-velero-operator/pull/91, and fixing ubi image to match repo

All the rehearsals should pass on this one

### CODE REVIEW

File: ci-operator/config/openshift/managed-velero-operator/openshift-managed-velero-operator-master.yaml [hunk 1]
```diff
@@ -1,24 +1,13 @@
 base_images:
   base:
-    name: ubi
+    name: ubi-minimal
```
Comment (2uasimojo) [inline ci-operator/config/openshift/managed-velero-operator/openshift-managed-velero-operator-master.yaml:3]: unrelated?

**Review by 2uasimojo — APPROVED** [review]
/lgtm

Also \o/

But you're going to need a MVO approver...

File: ci-operator/config/openshift/managed-velero-operator/openshift-managed-velero-operator-master.yaml (hunk 1 above)
Comment (cblecker) [reply ci-operator/config/openshift/managed-velero-operator/openshift-managed-velero-operator-master.yaml:3]: No, fixing oversight to match repo: https://github.com/openshift/managed-velero-operator/blob/6cb4cfec598c7fb80046c4f7ff4d2bb24f501e60/build/Dockerfile#L11

Comment (cblecker) [reply ci-operator/config/openshift/managed-velero-operator/openshift-managed-velero-operator-master.yaml:3]: Updated PR body to clearly call this out

**Comment by 2uasimojo** [comment]
> But you're going to need a MVO approver...

Oh, maybe not since you're an approver? One day I'll understand exactly how that works.
